src/aurelia-slickgrid/custom-elements/aurelia-slickgrid.ts

```diff
@@ -495,6 +500,12 @@ export class AureliaSlickgridCustomElement {
     this._eventPubSubService.publish(`${aureliaEventPrefix}onAureliaGridCreated`, aureliaElementInstance);
   }
 
+  /** This is an Aurlia lifecycle hook */
+  detaching() {
+    this.detached();
```

Comment:
is `detaching` a new lifecycle hooks that replaces `detached`? If so then we might as well just renamed `detached` to `detaching`, or is that something different?

Reply:
Yeah, `detaching` is a new hook. I left `detached` there because it's got a parameter which is not compatible with the new signature